src/renderers/shaders/ShaderChunk/aomap_fragment.glsl

```diff
@@ -1,5 +1,6 @@
 #ifdef USE_AOMAP
 
+	// reads channel R, intended to be used with a combined OcclusionRoughnessMetallic (RGB) texture
```

Comment:
// reads channel R, compatible with an Occlusion/Roughness/Metalness (RGB) texture

I would prefer to say it is "compatible". Saying it is "intended" sounds like a requirement.